src/mesh/NodeDB.cpp

```diff
@@ -958,6 +958,11 @@ void NodeDB::installRoleDefaults(meshtastic_Config_DeviceConfig_Role role)
         moduleConfig.telemetry.environment_update_interval = MAX_INTERVAL;
         moduleConfig.telemetry.air_quality_interval = MAX_INTERVAL;
         moduleConfig.telemetry.health_update_interval = MAX_INTERVAL;
+    } else if (role == meshtastic_Config_DeviceConfig_Role_CLIENT_BASE) {
```

Comment:
```
 } else if (IS_ONE_OF(role, meshtastic_Config_DeviceConfig_Role_ROUTER,
                       meshtastic_Config_DeviceConfig_Role_ROUTER_LATE, meshtastic_Config_DeviceConfig_Role_CLIENT_BASE)) {
```

for also covering 0-cost hop feature, which is also based on favorites 

https://github.com/meshtastic/firmware/pull/7992/files#diff-fb9b4a4da3229ad0f8213d3e55e97cb8068e71ba0225002aa050cf06788f6b8aR107